packages/frontend/src/components/common/Table/ScrollableTable/TableHeader.tsx

```diff
@@ -78,15 +83,11 @@ const TableHeader: FC<TableHeaderProps> = ({ minimal = false }) => {
                                                         withinPortal
                                                         maw={400}
                                                         multiline
-                                                        label={
-                                                            meta?.item &&
-                                                            isField(meta?.item)
-                                                                ? meta.item
-                                                                      .description
-                                                                : undefined
-                                                        }
+                                                        label={tooltipLabel}
                                                         position="top"
                                                         disabled={
+                                                            tooltipLabel ==
```

Comment:
perhaps this instead?
```suggestion
                                                            !tooltipLabel ||
```

in case it's `undefined`

Reply:
The soft-comparison will cover `undefined` here, but in retrospect the `== null` pattern isn't super common in the rest of the codebase, so this is a good suggestion regardless - will include it!